src/App/Application.cpp

```diff
@@ -2653,6 +2653,12 @@ void Application::initConfig(int argc, char ** argv)
                               mConfig["BuildVersionPoint"].c_str(),
                               mConfig["BuildVersionSuffix"].c_str(),
                               mConfig["BuildRevision"].c_str());
+
+        if (SafeMode::SafeModeEnabled()) {
+            Base::Console().Message("FreeCAD is running in _SAFE_MODE_.\n");
+            Base::Console().Message("Safe mode temporarily disables your configurations and "
+                                    "addons. Restart the application to exit safe mode.\n\n");

```

Comment:
You can combine these two messages into one call.
```suggestion
            Base::Console().Message("FreeCAD is running in _SAFE_MODE_.\n"
                                    "Safe mode temporarily disables your configurations and "
                                    "addons. Restart the application to exit safe mode.\n\n");

```